### PR TITLE
refine stability

### PR DESCRIPTION
**Pull request by weekface**
- refine logger
- fix scheduled backup case bug
- disable scheduled backup when finished
- fix TruncateSSTFile bug

### CODE REVIEW

File: tests/actions.go [hunk 1]
```diff
@@ -1369,14 +1370,12 @@ func (oa *operatorActions) cloneOperatorRepo() error {
...
-		cp -rf charts/tidb-operator %s &&
-		cp -rf charts/tidb-cluster %s &&
-		cp -rf charts/tidb-backup %s`,
-		oa.cfg.OperatorRepoDir, tagName, filepath.Join(oa.cfg.ChartDir, tagName), oa.operatorChartPath(tagName), oa.tidbClusterChartPath(tagName), oa.backupChartPath(tagName))
+	cmd := fmt.Sprintf("cd %s && git stash -u && git checkout %s && "+
+		"mkdir -p %s && cp -rf charts/tidb-operator %s && "+
+		"cp -rf charts/tidb-cluster %s && cp -rf charts/tidb-backup %s",
+		oa.cfg.OperatorRepoDir, tagName,
+		filepath.Join(oa.cfg.ChartDir, tagName), oa.operatorChartPath(tagName),
+		oa.tidbClusterChartPath(tagName), oa.backupChartPath(tagName))
```
Comment (shuijing198799) [inline tests/actions.go]: why use “+” in sprintf？

Comment (weekface) [reply tests/actions.go]: use `+` to join two or more string to a single string, this is normal usage.

**Pull request merged**